.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,7 +19,7 @@
 
 ### Before submitting the PR, please make sure you do the following
 
-- [ ] Read the [Contributing Guidelines](https://github.com/originjs/vite-plugin-federation/blob/main/CONTRIBUTING.md) and follow the [Commit Convention](https://github.com/originjs/vite-plugin-federation/blob/main/.github/commit-convention.md).
```

Comment:
I think it may be this. Could you change it to this address?
https://github.com/originjs/vite-plugin-federation/blob/main/CODE_OF_CONDUCT.md